index.html

```diff
@@ -104,8 +104,9 @@
         overlap: 1,
     });
 
-    //map, basePath, {ppm: int, saveURL: string, savePermission: bool, popoutUrl: string, assetName: string, hasLatewood: bool, initialData: json}
-    var treering = new LTreering(map, "", {'savePermission': true, 'saveURL': '/test', 'imagePath': '', 'ppm':
+    //map, basepath, {ppm: int, saveURL: string, savePermission: bool, popoutUrl: string, assetName: string, hasLatewood: bool, initialData: json}
+		//basepath is the local image folder location (i.e. replace with "file:///C:/Users/.../leaflet-treering")
+		var treering = new LTreering(map, basePath, {'savePermission': true, 'saveURL': '/test', 'ppm':
```

Comment:
Not sure why it's doing that. No matter how I change the code in Atom, these indents remain.

Reply:
basePath here should probably stay as "". There's no variable named basePath in the index.html file, so this is passing an undefined value. Javascript treats that a blank so it doesn't break things, but it could be a little confusing.